dcpy/connectors/socrata/extract.py

```diff
@@ -37,7 +39,10 @@ def get_version(source: Source):
     """For given socrata dataset, get date last updated formatted as a string.
     This is used as a proxy for a 'version' of the dataset."""
     url = get_metadata_url(source)
-    print(url)
     resp = requests.get(url)
     resp.raise_for_status()
     return _get_version_from_resp(resp.json())
+
+
+def download(source: Source, filename: str, dir: Path):
+    web.download_file(get_download_url(source), filename, dir)
```

Comment:
Small wrapper but just thought it made since, since eventually we might move away from using requests for socrata downloads (towards say one of their actual api)